### PR TITLE
Editing the final cave quest.

### PR DESCRIPTION
**Pull request by Renari**
I'm not sure what the original text means by: "It seems it speaks by watching you."

I changed it to the closest thing that seemed appropriate, within the context but that could be wrong.

### CODE REVIEW

**Comment by terrapin47** [comment]
Most of the original lines here are fine as is.  Other than personal preference, there's not much reason to apply synonyms and switch words around.  If we do this, then every time a new set of eyes sees lines there will be changes.

**Comment by Sunaries** [comment]
I agree with @zhouyu47. Some lines you've edited are wrong, because it takes out the original meaning of the Korean counterpart.

@HelpfulKoreanGirl What do you think of the changes compared to the original that can be found [here](https://raw.githubusercontent.com/Treeofsavior/EnglishTranslation/master/QUEST_kor.tsv)?

**Comment by terrapin47** [comment]
Good changes are on lines 2447, 2448, 2451 and 2452, keep those.
2448 - don't remove `nor the determination`.  It means that she doesn't feel any evil intent coming from the revelation.
"But, I don't feel any evil energy nor malicious intent.{nl}So then, this revelation is really Goddess Laima's... Good.{nl}"

**Comment by vtange** [comment]
Agree with @zhouyu47. You don't need to change anything on those first couple lines of QUEST. Believe us, those lines have been changed tens of times by various people, don't add to the problem.

You might want to just start a new PR with only lines 2447 - 2452 worked on for cleanliness

**Comment by Renari** [comment]
"2448 - don't remove nor the determination. It means that she doesn't feel any evil intent coming from the revelation."

That's not what the word determination means, to be determined is to have a goal and to pursue that goal relentlessly. 

The problem with a lot of these translations is they use very basic words as well as having tons of sentence fragments instead of full sentences and excess use of words that are meant to show magnitude (e.g. really, very - any English composition class this is one of the first thing they will tell you to avoid in written English).

**Comment by terrapin47** [comment]
You're right, that's not what it means, and that's why it should change.  It's a mistranslation.  I was saying that you should change it to the correct meaning, instead of just removing it because it's wrong.

**Comment by ttgmichael** [comment]
@Renari The word we are looking for to replace 'determination' is 'intent'; 

_However, I don't feel any evil energy nor intent at all._

I second @zhouyu47. 

I understand you want to contribute, but you can't just proofread by deleting things or modifying lines in such a way that strays them too far from the original Korean meaning.  All translated lines should be close/faithful to the Korean text. You can suggest better ways of saying things, but every thing (nouns/verbs) should be close to their original

prophesized - also basic English 101: don't use flowery vocab if you don't know what the difference is. 1. prophesized isn't the correct word,  prophesied is. To prophesy is to tell someone what the future beholds. Foresee is to see personally what the future beholds. Laima is telling the player what she saw, she isn't telling the player that she told someone what she saw.

_600 years ago, I foresaw a great deal of suffering and death_. Would work great without straying from the Korean text. 

_all the goddesses_ -> _all of the goddess(es)_ is unnecessary, they are both correct:
http://english.stackexchange.com/questions/80412/is-of-necessary-in-all-of

_I resolved to divide my being and subsist inside these many messages._ is fine already, your change strays from the Korean text

Also, how are 'resolve' and 'subsist' basic words?! Your changes made the line seem more basic than what it was. Note, I'm not saying your change is bad because it's basic, I'm saying the changes stray from the Korean text. **If anything, using basic words is suppose to be a good thing in English**, I'm not sure why you find using basic words a problem. You ought to take a few advice from a writer like George Orwell :)
http://www.pickthebrain.com/blog/george-orwells-5-rules-for-effective-writing/
**Never use a long word where a short one will do.**

_Messages that I continued to create secretly over a millennium._ I verified with the Korean text; good catch on millennia/millennium.

**QUEST_20150317_000021 actually feels already pretty off from the Korean text**, so you are proofreading a line has already strayed a bit from the Korean text (which is **not** what we want because otherwise, we get something like this. We **think** it's an accurate line, but proofreading it any further will more of less stray from the actual meaning).

I'll give a raw translation from Korean
_Savior, if you plan to help me regain my complete form and stop the demons' sinister plans, {nl} you must locate and bring together every revelations that I have concealed within the axes of time._

I'm not gonna go through any other lines in this PR, but I hope I got my point across. Please try and frame your revisions as close to the original meaning as possible. Sentence fragments could be because some of these lines aren't NPC dialogues. It could also be the case that the line was recently translated by IMC, but hasn't been looked at thoroughly. I'm not sure what you mean by "excess use of words that are meant to show magnitude", but I'll say that most of the lines are translated so that they retain the meaning of the Korean text. 

tldr; if you are proofreading, try minor changes that doesn't stray from the meaning of the line.

if you feel the meaning of the line is awkward, join us at Discord https://discordapp.com/channels/80317349386780672/80317349386780672
and we can discuss what should be changed for said line. **You should try using a Korean-English word dictionary to verify the meaning of the sentence before attempting to change it.**

**Comment by Renari** [comment]
prophesized - http://www.merriam-webster.com/dictionary/prophesize : to make a prediction

Prophesized vs prophesied is likely equivalent to using favourite vs favorite. One of these is considered correct in European English while the other is considered correct in American English.

> Also, how are 'resolve' and 'subsist' basic words?! Your changes made the line seem more basic than what it was. Note, I'm not saying your change is bad because it's basic, I'm saying the changes stray from the Korean text. If anything, using basic words is suppose to be a good thing in English, I'm not sure why you find using basic words a problem. You ought to take a few advice from a writer like George Orwell :)

Resolve and subsist are not basic words, however subsist is also not a common word. Using basic words is fine to an extent but you shouldn't be reading through lines of the game and consistently have them sound like a 5 year old wrote them. Like so:

> 600 years ago, I foresaw a lot of grief and death.
> 
> QUEST_20150317_000021 Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.

It's possible that axis was intended here. To bring something together is the same as collecting them thus we can condense the sentence and that retains the same meaning. 

> I will regain my complete form and help against the demons.

The word complete here is not required since it's implied that she is incomplete due to being separated into her many messages. Adding " be able to" maybe be the bit you are referring to as straying from the original meaning since using the phrase " be able to" does have a slight bit of uncertainty that was not present in the original. 

> Savior, if you plan to help me regain my complete form and stop the demons' sinister plans, {nl} you must locate and bring together every revelations that I have concealed within the axes of time.

This is event different from the translation that is currently there, thus it makes sense that what I edited is different from the Korean text. Because the text I edited was different from the Korean text. I'll change it to something more suitable.

**Comment by Renari** [comment]
> https://discordapp.com/channels/80317349386780672/80317349386780672

I can't join currently as I do not have a mic (currently working on that since my old headset had a speaker die).

**Comment by ttgmichael** [comment]
> Prophesized vs prophesied is likely equivalent to using favourite vs favorite. One of these is considered correct in European English while the other is considered correct in American English.

For this translation project, we are using American English here, so I will say "prophesied" would be the correct word to use by convention.

But still, prophesied is not the word to use. Laima isn't predicting anything, she **knows** for certain what'll happen. She **foresaw** the future. There's a difference and it's in the Korean text.

> Resolve and subsist are not basic words, however subsist is also not a common word

Just because subsist isn't a common word doesn't warrant it to be deleted. The verbs resolved and subsist are actually in the Korean text, so you are deleting details from the Korean text. Instead, if it feels too difficult, we can use simpler words like decide and embed. For example:

_I decided to divide my being and embed them inside these messages._

> To bring something together is the same as collecting them thus we can condense the sentence and that retains the same meaning.

Yes, "bring together" and "collecting" are similar, but just like how you see "a lot" and "a great deal" as two similar terms with different weights, I see "bring together" as a more vivid term. Collecting feels too general and I can't picture Laima asking you to collect her revelations (as if they are some sort of slate collection or a hobby), but rather bring them together as independent/separated objects.

> The word complete here is not required since it's implied that she is incomplete due to being separated into her many messages. 

Complete is not required, but in order to stay true to the Korean text, we chose to keep it. It doesn't break the flow of the sentence and it's not really redundant.  

> This is event different from the translation that is currently there, thus it makes sense that what I edited is different from the Korean text. Because the text I edited was different from the Korean text. I'll change it to something more suitable.

Well ofc if you are editting lines that have deviated from the Korean text, then your version is just as deviated. **But I'm also stressing that for some of the lines in your PR, you've actually editted lines properly translated from the Korean text and deviating from them.** We need to stay consistent with the Korean text as that is the only assurance to us that the English lines in the game matches the Korean text verbatim. There are ways to write these lines without having to create too many new words or introduce any new grammar structure. And you should be especially wary of deleting anything without checking with someone (or a dictionary) whether you have deleted bits of details from the Korean text. 

> I can't join currently as I do not have a mic (currently working on that since my old headset had a speaker die).

Actually you don't need a mic to join, we rarely use the chat function xD.

**Comment by Renari** [comment]
> I resolved to divide my being and subsist inside these many messages.

and 

> I divided my being inside of many messages.

Say the same exact thing in different words. The second being more commonly understood. I don't think embed is necessary but it is a fairly common term.

> I decided to divide my being between many messages.

As for `bring together` I haven't ever seen or heart this term used when not referring to gathering people. For objects you usually say you are gathering or collecting them.

**Comment by ttgmichael** [comment]
Hmm, QUEST_20150317_000019 is also straying from the actual meaning a bit, which correcting it may help you understand how Laima is conveying her message. It should be:

Raw translation:

> However, the demons would not stay still if they found out about my foresight. {nl}I had to conceal myself, until the time of your appearance.

Thus coming from that sentence, the two sentences are similar but one is missing nuance details.

> I resolved to divide my being and subsist inside these many messages/revelations.

Knowing she had to hide, Laima made the decision to divide herself and place them into her revelation. Implies that she had a choice on how to hide herself. Also note: she hasn't divided herself in this sentence yet, she only made the decision to.

> I divided my being inside of many messages.

Laima divides herself in this sentence. What's the problem here, you ask? Well read on to the next sentence. (I'll be taking a fixed original translation, yours is an incomplete sentence)

> _I continued to secretly create revelations while the millennium flew by the kingdom._

In your version, Laima has already divided but she **continued** to create messages through the millennium, which is a problem. How can she be dividing herself over messages that she hasn't created yet? You might say she is dividing her being as she continued to create messages. But I don't think the structure of your sentence lends to that interpretation because: 

> Messages that I continued to create secretly for many millennia.

Your second sentence says that these messages hasn't been **created** yet when she divided. 

> As for bring together I haven't ever seen or heart this term used when not referring to gathering people. For objects you usually say you are gathering or collecting them.

You mean you never heard "Bring/put together pieces of a puzzle?" xD Her revelations are more like pieces of a puzzle, and you simply don't collect pieces of a puzzle.

**Comment by HelpfulKoreanGirl** [comment]
Hey guys, sorry I'm late to the discussion! You've already acknowledged the problem that many of these changes are personal preference changes, and I do agree that if we're sticking as close as possible to the meaning in the Korean text, a lot of these proposed changes should not be made.  

QUEST_20150317_000016 Foresaw is more appropriate than prophesied because the goddess is saying she actually saw it, as in a vision that she had.

QUEST_20150317_000016 The Korean version of the text uses the plural form for goddesses, so that would be more appropriate than changing it to goddess.  The literal translation is closer to "the goddesses will have hidden all traces of their existence."

QUEST_20150317_000020 Inb4 by ttgmichael and his clear reasoning, but I'd like to add that just strictly based on literal translation, it does say she resolved to divide her being, not that she divided.

QUEST_20151001_002447 Surprising is a more accurate translation here than amazing. She is surprised because "this was definitely made several hundred years ago.. yet it appears to be speaking directly to your present self."

QUEST_20151001_002448 You need to keep the "But, " in there. Taking it out would probably distort the flow of the conversation.

I second Zhouyu47's translation of this: "But, I don't feel any evil energy nor malicious intent.{nl}So then, this revelation is really Goddess Laima's... Good.{nl}" 

Sorry to say this, but all the other changes really seem unnecessary. :/

**Comment by Renari** [comment]
> You mean you never heard "Bring/put together pieces of a puzzle?" xD Her revelations are more like pieces of a puzzle, and you simply don't collect pieces of a puzzle.

Nope, when you're telling someone you are doing a puzzle you usually say "I'm doing a puzzle" or "I'm putting a puzzle together." not "I'm bringing together a puzzle."

@HelpfulKoreanGirl 

> QUEST_20150317_000016

We already went over that.

> QUEST_20150317_000017

This as well, and I changed it back.

> QUEST_20150317_000020

The definition for resolve may make sense here but people don't speak like this. The word resolved used in context is normally followed by an issue. "I resolved the problem." "We're working on resolving the case." If you want to show that this isn't something she decided but rather something that was forced upon her then you would simply say that she was forced.

> QUEST_20151001_002447

This is another example of something that isn't normally used. The word surprising is fairly common but not used on it's own like this. 

> QUEST_20151001_002448

Yes I've already changed this, the word malicious was not in my changes however the "... Good" at the end of this sentence is very awkward and if it's going to still be used it should be moved to the beginning of the sentence.

**Comment by HelpfulKoreanGirl** [comment]
QUEST_20151001_002448 Good can be replaced with "... I see."

File: QUEST.tsv [hunk 1]
```diff
@@ -13,15 +13,15 @@ QUEST_20150317_000012	Bokor Master
...
-QUEST_20150317_000021	Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.
-QUEST_20150317_000022	This divine task may be a heavy burden, but it is something you must bear. {nl}Perhaps, in the muddy midst of your trials and tribulations, something may come about.
-QUEST_20150317_000023	Nevertheless, only you and I can save this world.
-QUEST_20150317_000024	I've passed the next Revelation to the first Paladin. {nl}I'll be waiting for you at the highest flower garden.
+QUEST_20150317_000019	However, knowing of my clairvoyance, the demons would not ignore what I had seen. {nl}I was forced to hide, until the time of your appearance.
+QUEST_20150317_000020	I decided to divide my being inside of many messages. {nl}Messages that I continued to create secretly for many millennia.
+QUEST_20150317_000021	Savior, if you will help me regain my form we can stop the demons sinister plans.{nl}You must collect the revelations that I have hidden throughout many axis of time.
```
Comment (Sunaries) [inline QUEST.tsv]: demons -> demons'

File: QUEST.tsv [hunk 2]
```diff
@@ -2436,13 +2436,13 @@ QUEST_20150918_002443	There is a box over there. Shall we check it?
...
-QUEST_20151001_002451	You better not tell the world about it.{nl}If you don't want the demons to go look for you.{nl}
-QUEST_20151001_002452	The Revelation doesn't want me to mention about the savior, I guess.{nl}Okay, please go back to meet Uska.{nl}
-QUEST_20151001_002453	He will tell you where to go.{nl}May the Goddess bless you in the future.
+QUEST_20151001_002447	Amazing.{nl}It must have been created hundreds of years ago..{nl}It seems it speaks of you.{nl}
+QUEST_20151001_002448	I don't feel any evil energy nor malicious intent.{nl}Then, this revelation must have come from the Goddess Laima...{nl}
+QUEST_20151001_002449	I'm suprised about this talk of a savior.{nl}
+QUEST_20151001_002450	Amongst the many Revelators, the one who can save the world and the Goddesses...{nl}The revelation says you are the savior.{nl}
+QUEST_20151001_002451	You better keep quiet about this.{nl}Unless you want the demons looking for you.{nl}
+QUEST_20151001_002452	The revelation doesn't want me to mention anything about the savior.{nl}Okay, please hurry back to Uska.{nl}
+QUEST_20151001_002453	He'll know where to go.{nl}May the Goddess bless you.
```
Comment (Sunaries) [inline QUEST.tsv:2445]: "He'll know where to go" is ambiguous. "He will tell you where to go" is fine and clearer and doesn't need to be changed. 

Why did you remove 'in the future'? It's there.

File: QUEST.tsv [hunk 3]
```diff
@@ -13,15 +13,15 @@ QUEST_20150317_000012	Bokor Master
 QUEST_20150317_000013	So, you want to learn more about this stone slate. {nl}If you wish, I shall show you the will of its creator.
 QUEST_20150317_000014	Goddess Laima
 QUEST_20150317_000015	By the time you get this message, much time will have passed. {nl}I am the Goddess of Fate and Wisdom, Laima.
-QUEST_20150317_000016	600 years ago, I foresaw a lot of grief and death. {nl}If you're seeing this message, then I'm afraid all I've foreseen has likely come to pass.
-QUEST_20150317_000017	The demons have started the first of the cataclysms, and all the Goddesses have vanished. {nl}I knew this would happen and did all I could to change this fate, but, my power alone was not enough.
+QUEST_20150317_000016	600 years ago, I foresaw a great deal of death and suffering. {nl}If you're seeing this, then I'm afraid what I foresaw has come to pass.
+QUEST_20150317_000017	The demons have started the first of the cataclysms, and all of the goddesses have vanished. {nl}I knew this would occur and did everything I could to prevent this fate, but my power was not enough.
```
Comment (Sunaries) [inline QUEST.tsv]: goddesses -> Goddesses

File: QUEST.tsv [hunk 4]
```diff
@@ -2436,13 +2436,13 @@ QUEST_20150918_002443	There is a box over there. Shall we check it?
...
 QUEST_20150918_002445	Is that a wing of a Goddess Statue over there?
 QUEST_20150918_002446	Huh? How did you get up here?
-QUEST_20151001_002447	Amazing.{nl}It must be created hundreds of years ago..{nl}It seems it speaks by watching you.{nl}
-QUEST_20151001_002448	But, I don't feel any evil energy nor the determination.{nl}So then, this Revelation is really of Goddess Laima's... That's right.{nl}
-QUEST_20151001_002449	We are more surprised about the savior.{nl}
-QUEST_20151001_002450	Among many Revelators, the one who can save this world and the Goddesses...{nl}The Revelation says that you are the savior.{nl}
-QUEST_20151001_002451	You better not tell the world about it.{nl}If you don't want the demons to go look for you.{nl}
-QUEST_20151001_002452	The Revelation doesn't want me to mention about the savior, I guess.{nl}Okay, please go back to meet Uska.{nl}
-QUEST_20151001_002453	He will tell you where to go.{nl}May the Goddess bless you in the future.
+QUEST_20151001_002447	Amazing.{nl}It must have been created hundreds of years ago..{nl}It seems it speaks of you.{nl}
```
Comment (Sunaries) [inline QUEST.tsv]: Just like @HelpfulKoreanGirl said, surprise is more accurate. 

Amazing -> I'm surprised.

**Comment by Renari** [comment]
> "He'll know where to go" is ambiguous. "He will tell you where to go" is fine and clearer and doesn't need to be changed.

I did this because it makes the sentence seem as if she is ordering you to go talk to him. In this phrasing the makes her sound less bossy and it gives the impression that the person you are going to has the information you want. It doesn't sound like your forcing the player to go talk to x npc next rather implying that this is a good place to go. This gives a sense of individuality that is good for game dialog.

> Why did you remove 'in the future'? It's there.

Would you say "God bless you in the past?" The fact that she is saying the phrase at all implies that she is referring to the future.

File: QUEST.tsv (hunk 2 above)
Comment (ttgmichael) [inline QUEST.tsv:2445]: May the Goddess bless your future/fate.

And I agree with Spadow; He'll know where to go is very ambiguous. It could mean someone knows where he needs to go. It doesn't always imply that he'll guide the player to the location the player needs to go.

File: QUEST.tsv (hunk 3 above)
Comment (ttgmichael) [inline QUEST.tsv:17]: goddess -> Goddess

File: QUEST.tsv (hunk 1 above)
Comment (ttgmichael) [inline QUEST.tsv]: I'm still not pleased to use the word 'collect', it's a word that you'll see all the time in other quests. It almost makes the word sound trivial.

You could say:

> You must recover the revelations that I have hidden throughout the axis of time.
